src/components/ButtonPanel.js

```diff
@@ -0,0 +1,36 @@
+import Button from './Button';
+import './ButtonPanel.css';
+
+const buttonList = [
+  { label: 'AC', class: 'btn gray' },
+  { label: '+/-', class: 'btn gray' },
+  { label: '%', class: 'btn gray' },
+  { label: '/', class: 'btn orange' },
+  { label: '7', class: 'btn' },
+  { label: '8', class: 'btn' },
+  { label: '9', class: 'btn' },
+  { label: 'x', class: 'btn orange' },
+  { label: '4', class: 'btn' },
+  { label: '5', class: 'btn' },
+  { label: '6', class: 'btn' },
+  { label: '-', class: 'btn orange' },
+  { label: '1', class: 'btn' },
+  { label: '2', class: 'btn' },
+  { label: '3', class: 'btn' },
+  { label: '+', class: 'btn orange' },
+  { label: '0', class: 'btn zero' },
+  { label: '.', class: 'btn' },
+  { label: '=', class: 'btn orange' },
+];
+
+const ButtonPanel = () => (
+  <div className="ButtonPanel">
+    {buttonList.map((button) => (
+      <Button key={button.label} value={button.label} className={button.class}>
+        {button.label}
+      </Button>
+    ))}
+  </div>
+);
```

Comment:
- [ ] Kindly follow the given design for the calculator as this is a core requirement.

Your design
![image](https://github.com/Ruthmy/math-magicians/assets/81932478/7d608940-e176-4f8b-ae88-9523b42a9bbb)

Given design
![image](https://github.com/Ruthmy/math-magicians/assets/81932478/df71e9ad-9c18-469c-a4fe-46ed2f956685)

Reply:
Hello @Munsa1 !
Thanks for your feedback. 
In the requirements there is no design to follow, there is a medium-fidelity wireframes, but I recognize that I took several liberties and with them came an overflow, so I modified the base structure to correct it.
Thank you very much again.